### PR TITLE
docs: change to new using toolkit directive

### PR DESCRIPTION
**Pull request by zetashift**

### CODE REVIEW

**Comment by TonioGela** [comment]
Have I maybe asked you to do that recently? I was thinking of implementing it while reviewing the last PR you did :D

**Comment by zetashift** [comment]
> Have I maybe asked you to make this change recently
Yup, but wanted to change it after the docs PR because it'd be too many things at once :P

File: docs/examples.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This example was written by [Koroeskohr] and taken from the [Virtuslab Blog](htt
 @:choice(scala-3)
 
 ```scala mdoc:silent
-//> using lib "org.typelevel::toolkit::@VERSION@"
+//> using toolkit typelevel::@VERSION@
```
Comment (armanbilge) [inline docs/examples.md:13]: Actually it should be a single colon. Does the `::` work? I'm not sure 🤔

Comment (zetashift) [reply docs/examples.md:13]: Yeah it compiled an example just fine :P :
```
❯ scala-cli run .
Compiling project (Scala 3.3.1, JVM (19))
Compiled project (Scala 3.3.1, JVM (19))
Finished writing books to books.csv.
```
I'll make the PR to fix the colon!

**Pull request merged**